metrics/opentelemetry.go

```diff
@@ -108,22 +118,30 @@ var (
 		Measure:     BytesTransferredTotal,
 		Aggregation: view.Sum(),
 	}
-	bitswapRequestWithIndexerCandidatesView = &view.View{
-		Measure:     BitswapRequestWithIndexerCandidatesCount,
+	failedRetrievalsPerRequestView = &view.View{
+		Measure:     FailedRetrievalsPerRequestCount,
+		Aggregation: view.Distribution(0, 1, 2, 3, 4, 5, 10, 20, 40),
+	}
+	requestWithIndexerCandidatesFilteredView = &view.View{
+		Measure:     RequestWithIndexerCandidatesFilteredCount,
 		Aggregation: view.Count(),
 	}
-	bitswapRequestWithSuccessfulQueryView = &view.View{
-		Measure:     BitswapRequestWithSuccessfulQueryCount,
+	requestWithIndexerCandidatesView = &view.View{
+		Measure:     RequestWithIndexerCandidatesCount,
 		Aggregation: view.Count(),
 	}
-	indexerCandidatesView = &view.View{
-		Measure:     IndexerCandidatesCount,
+	requestWithSuccessfulQueriesFilteredView = &view.View{
+		Measure:     RequestWithSuccessfulQueriesFilteredCount,
 		Aggregation: view.Count(),
 	}
-	retrievalQueryView = &view.View{
-		Measure:     RetrievalQueryCount,
+	requestWithSuccessfulQueriesView = &view.View{
+		Measure:     RequestWithSuccessfulQueriesCount,
 		Aggregation: view.Count(),
 	}
+	indexerCandidatesPerRequestView = &view.View{
+		Measure:     IndexerCandidatesPerRequestCount,
+		Aggregation: view.Distribution(0, 1, 2, 3, 4, 5, 10, 20, 40),
```

Comment:
👌 does the `40` bucket end up collecting `>=40`? I doubt there's any of these even in the indexer at the moment so I'm mainly interested in how this works.

Reply:
Based on the [description of the `view.Distribution` function](https://pkg.go.dev/go.opencensus.io/stats/view?utm_source=godoc#Distribution), I believe there is going to be one more bucket for 40 -> +infinity.